project2/style.css

```diff
@@ -0,0 +1,64 @@
+.card {
+    background-color: rgb(103, 129, 138);
+    color: white;
+    padding: 20px;
+    width: 640px;
+}
+
+img {
+    max-width: 100%;
+}
+
+h1 {
+    font-size: 65px;
+    position: relative;
+    top: 25px;
+    text-align: center;
+}
+
+h2 {
+    font-size: 20px;
+    text-align: center;
+}
+
+h3 {
+    font-size: 12px;
```

Comment:
16px is usually the default font size for browsers; going lower for that, especially for something users may read from a distance, is not recommended.